lib/stackprof.rb

```diff
@@ -1,7 +1,7 @@
 require "stackprof/stackprof"
 
 module StackProf
-  VERSION = '0.2.15'
+  VERSION = '0.2.16'
```

Comment:
Generally speaking, I _**think**_ the version bumps have been done outside feature PRs in this repo.  Not positive on that statement, but something to consider.

Reply:
I "brought" this practice from a different repository, but you are right.